calicoctl/commands/node/run.go

```diff
@@ -256,6 +274,58 @@ Description:
 		fmt.Println(err)
 		os.Exit(1)
 	}
+
+	// Create the command to follow the docker logs for the calico/node
+	logCmd := exec.Command("docker", "logs", "calico-node", "--follow")
+
+	// Get the stdout pipe
+	outPipe, err := logCmd.StdoutPipe()
+	if err != nil {
+		fmt.Println("Error querying calico/node logs:")
+		fmt.Println(err)
+		os.Exit(1)
```

Comment:
Well I don't expect it to fail, so an error querying the logs is an odd thing to occur and certainly something is not working correctly.